### PR TITLE
Cleanup prober to be similar to other tools.

### PR DESCRIPTION
**Pull request by pmtischler**
It used a different program structure and argument names.

### CODE REVIEW

File: client/tools/prober.py [hunk 1]
```diff
@@ -13,32 +14,37 @@
...
     Args:
-        client: The interop client.
-        data_generator: The generator of UAS telemetry.
-        interop_time: The time between interoperability execution.
-        logger: The logger to write status to.
+        url: The interoperability URL.
+        username: The interoperability username.
+        password: The interoperability password.
+        generator: The data generator name to use.
+        flightsim_kml_path: The KML path to use if flightim generator.
```
Comment (prattmic) [inline client/tools/prober.py]: flightsim?

File: client/tools/prober.py [hunk 2]
```diff
@@ -61,47 +66,30 @@ def main():
...
+                        required=True,
+                        help='Username for interoperability.')
+    parser.add_argument('--password', help='Password for interoperability.')
+    parser.add_argument('--interop_time',
+                        type=float,
+                        help='Time between sent requests (sec).')
+    parser.add_argument('--generator',
                         type=str,
                         choices=['zeros', 'flightsim'],
                         help='Data generation implementation.')
```
Comment (prattmic) [inline client/tools/prober.py]: default=zeros perhaps?

**Comment by prattmic** [comment]
LGTM

File: client/tools/prober.py (hunk 1 above)
Comment (pmtischler) [inline client/tools/prober.py]: Done.

File: client/tools/prober.py (hunk 2 above)
Comment (pmtischler) [inline client/tools/prober.py]: Done.

**Pull request merged**